.sauce/cypress-10.yml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: cypress
+sauce:
+  region: us-west-1
+  concurrency: 10
+  metadata:
+    tags:
+      - e2e
+    build: "$BUILD_ID"
+rootDir: tests/e2e/cypress_10
+defaults:
+  mode: sauce
+  timeout: 5m
+docker:
+  fileTransfer: mount
+cypress:
+  version: 10.1.0
+  configFile: "cypress.config.js"
+suites:
+  - name: "chrome test"
+    browser: "chrome"
+    platformName: "macOS 11.00"
+    config:
+      env:
+        hello: world
+      specPattern: [ "cypress/e2e/**/*.js" ]
```

Comment:
Btw. did you consider putting this under `e2e` instead? If yes/no, what are your thoughts on it? Considering that this is the new grouping by cypress. `e2e` vs `component` --> https://docs.cypress.io/guides/references/configuration#e2e

Reply:
I checked the `e2e` and `component` config and I think there are two group of settings for `cypress.config.js/ts/...`.  I thought we should introducing two sets of config in saucectl config like 
```
type SuiteConfig struct {
  E2E E2E
  Component Component
}
type E2E struct {
  SpecPattern []string
  ExcludeSpecPattern []string
}
type Component struct {
  SpecPattern []string
  ExcludeSpecPattern []string
}
```
While checking the runner code, I found we are using the config for `cypress run`, which doesn't contain e2e and component https://docs.cypress.io/guides/guides/command-line#cypress-run-config-lt-config-gt
My thoughts are keeping the `specPattern`/`excludeSpecPattern` and letting user to set `e2e` or `component` in `cypress.config.js/ts/...`

Reply:
I checked the `e2e` and `component` config and I think there are two group of settings for `cypress.config.js/ts/...`.  I thought we should introducing two sets of config in saucectl config like 
```
type SuiteConfig struct {
  E2E E2E
  Component Component
}
type E2E struct {
  SpecPattern []string
  ExcludeSpecPattern []string
}
type Component struct {
  SpecPattern []string
  ExcludeSpecPattern []string
}
```
Just found I should also update the cypress runner for this. The config file is mentioned for a component spec file for cypress run https://docs.cypress.io/guides/guides/command-line#cypress-run

Let me update that a bit

Reply:
According to the discussion in standup, added `testingType` for cypress 10 to indicate test type.